gui/src/style.css

```diff
@@ -505,12 +520,19 @@ body {
     bottom: 90px;
     left: 50%;
     transform: translateX(-50%);
-    display: none;

```

Comment:
It just fixes the copied popup and breaks the "You didn't select the theme."

Reply:
how exactly does it breaks? can you share a screenshot?

Reply:
The "You didn't select the theme." popup is shown with `display: block;` instead of `visibility: hidden;` and `opacity: 0;`. Since you removed 'display: none;' and didn't write any JavaScript to change `visibility` or `opacity`, the popup can't be made visible.

Reply:
oh ye, one sec